src/scss/06_components/layout/_layout.scss

```diff
@@ -43,15 +42,37 @@ $-sidebar-width: 340px;
       grid-template-rows: auto 1fr;
     }
   }
+}
 
-  @include breakpoint($breakpoint-xl) {
-    column-gap: $spacing-triple;
+.has-sidebar .main-content {
+  @include breakpoint($breakpoint-m) {
+    @include components-container-max-width;
   }
 }
 
-.has-sidebar .main-content {
+// Set proper width for all generic page components
+.page-title,
+.lead-in,
+.page-divider,
+.content-date,
+.main-image,
+.long-desc,
+.enriched-content:not(.components),
+.content-links,
+.content-tags:not(.content-tags--card),
+.social-media__items {
```

Comment:
Should we maybe consider some helper class for these in the future?

Reply:
Yes, definitely! Added TODO with a ticket number.